tests/scripts/setup_test

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env bash
+set -e
+
+# remove tls cert for oidc login
+FINGERPRINT=$(lxc config trust list | grep lxd-ui.crt | awk '{print $8}')
+lxc config trust remove ${FINGERPRINT}
+
+# setup oidc configs
+if [ -f .env.local ]
+then
+  set -o allexport; source .env.local; set +o allexport
+fi
```

Comment:
It is now mandatory to create the .env.local file to run the e2e tests locally. Can you document how to get the values and which config values are needed in the HACKING.md section about e2e tests?

Reply:
Yes, I commented on the PR that this is a TODO. Just wanted you to have a look at the PR first in case if anything needed to change regarding the approach. Will add to HACKING.md.

Reply:
I've added instructions for setting up oidc to run the e2e tests with reference to the [oidc setup wiki page](https://github.com/canonical/lxd-ui/wiki/Setup-oidc-login). Let me know if this is sufficient?